v21.2/connection-reference.md

```diff
@@ -0,0 +1,570 @@
+---
+title: Client Connection Reference
+summary: How to connect to a CockroachDB cluster from different tools
+toc: true
+---
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="serverless">{{ site.data.products.serverless-plan }}</button>
+  <button class="filter-button page-level" data-scope="dedicated">{{ site.data.products.dedicated }}</button>
+  <button class="filter-button page-level" data-scope="core">{{ site.data.products.core }}</button>
+</div>
+
+This page documents the required connection configuration for each [fully-supported, third-party tool](third-party-database-tools.html).
+
+For a list of all supported cluster connection parameters, see the [`cockroach` Connection Parameters](connection-parameters.html).
+
+<div class="filters clearfix">
+<button class="filter-button page-level" data-scope="js-ts">JavaScript/TypeScript</button>
+  <button class="filter-button page-level" data-scope="python">Python</button>
+  <button class="filter-button page-level" data-scope="go">Go</button>
+  <button class="filter-button page-level" data-scope="java">Java</button>
+  <button class="filter-button page-level" data-scope="ruby">Ruby</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="js-ts">
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="node-postgres">node-postgres</button>
+  <button class="filter-button page-level" data-scope="sequelize">Sequelize</button>
+  <button class="filter-button page-level" data-scope="typeorm">TypeORM</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="node-postgres">
+
+## Connect with node-postgres
+
+To connect to CockroachDB with [node-postgres](https://node-postgres.com), create a new [`Client`](https://node-postgres.com/api/client) object with a connection string.
+
+For example:
+
+{% include copy-clipboard.html %}
+~~~ js
+const { Client } = require('pg')
+
+const connectionString = '<connection-string>'
```

Comment:
I'm concerned that this might encourage developers who don't know any better to paste their connection strings into their source code, which is sketch because it contains the password. What do you think of modifying these examples to instead read from an environment variable named `DATABASE_URL`?